gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -512,7 +518,12 @@ function roleselection.GetSelectableRolesList(maxPlys, rolesAmountList)
 			selectableRoles[subrole] = rolesAmountList[subrole]
 
 			curRoles = curRoles + 1
+
+			break
```

Comment:
Line 483 still has the loop, it looks like you added break here but you could just remove the for loop because you will always hit this break regardless

Reply:
Yes I can, as I said, I tried to change as minimal as possible to see if thats enough. As the for-loop works yet.